wizard_builder/tests/test_sites.py

```diff
@@ -1,10 +1,11 @@
 from mock import patch
-from wizard_builder.models import PageBase, QuestionPage, SingleLineText
 
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.test import TestCase, override_settings
 
+from ..models import PageBase, QuestionPage, SingleLineText
```

Comment:
so I moved the tests inside the wizard builder folder so the tests could all be using relative imports, completely removing the possibility of the code accidentally running the wrong wizard builder version